funcs_scripts.php

```diff
@@ -145,6 +152,12 @@ function module_is_recipe()
  */
 function is_module()
 {
+    global $GITHUB_REF;
+    $repoData = MetaData::getMetaDataForRepository($GITHUB_REF);
+    if (isset($repoData['type'])) {
+        return $repoData['type'] !== 'other';
```

Comment:
This would return true if $type === 'recipe'

Reply:
Yes, which matches the `$moduleTypes` check below.